u3-views/views/partials/video-tease.hbs

```diff
@@ -0,0 +1,7 @@
+<a class='next-card' href='/videos/{{tease.id}}'>
+  <div><img src='/images/{{tease.id}}.webp' /></div>
+  <div class='description'>
+    <p>{{tease.title}}</p>
+    <p class='stats'>{{tease.views}} views · {{tease.date}}</p>
+  </div>
+</a>
```

Comment:
ดี! ฉันชอบที่สิ่งนี้ใช้ชื่อตัวแปรอื่น
Good! I like that this uses a different variable name.